package.json

```diff
@@ -17,7 +17,11 @@
     "test": "cs test --verbose --coverage",
     "cozyPublish": "git fetch origin ${DEPLOY_BRANCH:-build}:${DEPLOY_BRANCH:-build} && cs publish --token $REGISTRY_TOKEN --build-commit $(git rev-parse ${DEPLOY_BRANCH:-build})",
     "stack:docker:dev": "cs stackDocker",
-    "stack:docker:prod": "cs stackDocker --prod"
+    "stack:docker:prod": "cs stackDocker --prod",
+    "fixtures:drop": "ACH drop io.cozy.timeseries.geojson.aggregate io.cozy.timeseries.geojson",
+    "fixtures:accounts": "ACH import fixtures/accounts.json",
+    "fixtures:datas": "ACH script fixtures/importGeojson.js",
```

Comment:
You could simply add cozy-ach in devDep and do `"fixtures:datas": "yarn run ACH script  timeseries/importGeojson"` and avoid copy/pasting the importGeojson.js ;)

Reply:
By doing so, the  fixtures/geojson.json  file probably becomes not useful